dagshub/data_engine/model/datasource.py

```diff
@@ -120,7 +129,14 @@ def to_dict(self, ds: "Datasource") -> Dict[str, Any]:
 
 
 class Datasource:
-    def __init__(self, datasource: "DatasourceState", query: Optional[DatasourceQuery] = None, select=None, as_of=None):
+    def __init__(
+        self,
+        datasource: "DatasourceState",
+        query: Optional[DatasourceQuery] = None,
+        select: Optional[List[Dict]] = None,
```

Comment:
Shouldn't the `Dict` be typed?

Reply:
the typing is just a suggestion, it's a str/dict dict so it won't really be helpful